### PR TITLE
IA-2846 target java11

### PR DESCRIPTION
**Pull request by Qi77Qi**
https://broadworkbench.atlassian.net/browse/IA-2846

**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

**Review by kshakir — APPROVED** [review]
Changes LGTM

- TIL despite the attempt to move to semver [circa 2005](https://en.wikipedia.org/wiki/Java_version_history) some tools are still calling Java `1.x`
- DSP-BT has been asked to create tickets for even the smallest GitHub PRs for auditing purposes (CLIA? FedRAMP?). Not sure if that applies to other teams too?

File: project/Settings.scala [hunk 1]
```diff
@@ -118,7 +118,7 @@ object Settings {
   //common settings for all sbt subprojects
   val commonSettings = commonBuildSettings ++ commonTestSettings ++ List(
     organization := "org.broadinstitute.dsde.workbench",
-    scalaVersion := "2.13.5",
+    scalaVersion := "2.13.6",
```
Comment (Qi77Qi) [inline project/Settings.scala:121]: this fixes an issue in current `develop` branch that's preventing `TRAVIS-REPLACE-ME` being updated properly

**Review by rtitle — APPROVED** [review]
Looks good to me. Was debating with myself whether this deserves a version bump but I think this is fine.

**Comment by Qi77Qi** [comment]
@rtitle there shouldn't be any api change...so I think it's fine to not bump version

**Pull request merged**